ios/BrightcoveIMAPlayerView.m

```diff
@@ -106,10 +95,21 @@ - (void)setupWithSettings:(NSDictionary*)settings {
     _playbackController.autoPlay = autoPlay;
     _playbackController.allowsExternalPlayback = allowsExternalPlayback;
 
+    _playerView = [[BCOVPUIPlayerView alloc] initWithPlaybackController:self.playbackController options:nil controlsView:[BCOVPUIBasicControlView basicControlViewWithVODLayout] ];
```

Comment:
compared with the sample at BrightCove/BC  website, I'm quite surprised that previous version used slightly different syntax to initiate the instance, maybe BC SDK had upgraded later on, so sample had been updated as well,
as long as the new syntax works.